spec/models/user_spec.rb

```diff
@@ -177,19 +178,55 @@
       end
     end
 
-    describe "for inactive users" do
-      subject{ described_class.new active: false }
+    describe 'with an existing technical user' do
+      subject do
+        organization = create(:organization)
+        create(:technical_user, name: 'tech', organization: organization)
```

Comment:
I prefer declaring such data in `let` blocks, even if I only use it in the `subject`, because it makes the `subject` less complex and easier to read and it makes things potentially easier when you add future tests (which I suggested doing in the previous comment).